gradlew

```diff
@@ -0,0 +1,232 @@
+#!/bin/sh
```

Comment:
Is there a reason why you pushed this Gradle code & jar to the repository?

Reply:
github actions and artifact builds

Reply:
referring to https://github.com/Ascynx/UnofficialMonumentaMod/actions. It has the artifact when built so it's kinda neat to have.

Reply:
well that sure is nice :)